src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/UpgradeForm.tsx

```diff
@@ -103,17 +103,17 @@ function UpgradeForm({ selectedPlan, setSelectedPlan }: UpgradeFormProps) {
         <span>{owner}</span>
       </div>
       <PlanTypeOptions
-        setFormValue={setFormValue}
+        setFormValue={setFormValue as UseFormSetValue<UpgradeFormFields>}
```

Comment:
Instead of casting here, are we able to leverage the generics passed to `useForm`, in the docs they show something like this:

```ts
const {} = useForm<FormValues>({ })
```

Reply:
lemme see if I can make this work 🤔

Reply:
Changed in [de6acd7](https://github.com/codecov/gazebo/pull/3461/commits/de6acd7ef4aed2f2df06ca0e529819916fb4fb03)